src/CoreWCF.CrankTests/CoreWCFPerfService/CoreWCF.PerfService.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net472</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore" Version="2.2.0" />
+    <PackageReference Include="CoreWCF.Http" Version="1.1.0" />
```

Comment:
This is only needed for net472. For all other tfm's, you need to use `<FrameworkReference Include="Microsoft.AspNetCore.App" />`. Also this should be 2.1.x. I think the current version is 2.1.7. AspNetCore 2.2 is not in support, but 2.1 is still in support on net472 only. See [https://github.com/CoreWCF/CoreWCF/blob/main/src/CoreWCF.Http/tests/CoreWCF.Http.Tests.csproj](https://github.com/CoreWCF/CoreWCF/blob/main/src/CoreWCF.Http/tests/CoreWCF.Http.Tests.csproj) for reference.

Reply:
Have update